internal/rbd/rbd_util.go

```diff
@@ -77,6 +77,13 @@ const (
 	// these are the migration label key and value for parameters in volume context.
 	intreeMigrationKey   = "migration"
 	intreeMigrationLabel = "true"
+
+	// these are the migration volume handle identifiers.
+	migHandleLength   = 4
+	migHandleFieldSep = "_"
+	migVersion        = "migrate-alpha1"
+	migImageNamePfx   = "kubernetes-dynamic-pvc-"
+	migMonPfx         = "mons-"
```

Comment:
are these are kubernetes defined one?  please use full names `migMonPfx` to `migratedMonitorPrefix` etc

Reply:
Do we have any documentation for these consts in kubernetes?

Reply:
Yep, added details. done.